### DISCUSSION TITLE
ReactNativeTVTypes.d.ts error

### DISCUSSION THREAD

**Discussion by kosso** [react-native-tvos/react-native-tvos#720] [Feature Requests]
Hi.
I've just come across a small issue with the typings for the ReactNativeTVTypes HWEvent names `velocityx` and `velocityy`

They should be camel cased as `velocityX` and `velocityY`. 

You can see the actual names by logging the `body.velocityY` and `body.velocityX` output from the `useTVEventHandler` event while panning around the 'black' Apple TV remote control pad. 

Here's the location of the issue: 
See: https://github.com/react-native-tvos/react-native-tvos/blob/efc6306fa372a2368c2fb74faf9001db5e8da302/packages/react-native/types/public/ReactNativeTVTypes.d.ts#L57

You can see them camel-cased in all other places on both platforms here : 
https://github.com/search?q=repo%3Areact-native-tvos%2Freact-native-tvos+velocityx+velocityy&type=code

----

I wasn't really able to submit this as an actual 'Bug' since it's a typings issue that reveals itself in the IDE, so can't provide a reproducible. :)

**Comment by douglowder** [comment]
Fixed in #733 .
